component/core/src/main/java/org/exoplatform/social/core/storage/cache/SocialStorageCacheService.java

```diff
@@ -129,7 +132,9 @@ public ExoCache<IdentityFilterKey, IntegerData> getCountIdentitiesCache() {
   public ExoCache<ListIdentitiesKey, ListIdentitiesData> getIdentitiesCache() {
     return identitiesCache;
   }
-
+  public ExoCache<IdentityKey, Long> getProfileAvatarCache() {
+    return profileAvatarCache;
+  }
```

Comment:
Formatting, line break

```suggestion
  }

```

Reply:
fixed